compiler-core/src/language_server/tests/snapshots/gleam_core__language_server__tests__action__qualified_to_unqualified_import_function.snap

```diff
@@ -0,0 +1,21 @@
+---
+source: compiler-core/src/language_server/tests/action.rs
+expression: "\nimport result\n\npub fn main() {\n  result.is_ok()\n}\n"
+---
+----- BEFORE ACTION
+
+import result
+
+pub fn main() {
+           ↑   
+  result.is_ok()
```

Comment:
This code action should never be offered for functions. Unqualified functions are not supported.